app/components/custom_list/user_list_row/user_list_row.js

```diff
@@ -63,87 +65,83 @@ export default class UserListRow extends React.PureComponent {
             }, {displayname: usernameDisplay});
         }
 
+        const teammateDisplay = displayUsername(user, teammateNameDisplay);
+        const showTeammateDisplay = teammateDisplay !== username;
+
         return (
-            <CustomListRow
-                id={id}
-                theme={theme}
-                onPress={this.onPress}
-                enabled={enabled}
-                selectable={selectable}
-                selected={selected}
-            >
-                <ProfilePicture
-                    userId={id}
-                    size={32}
-                />
-                <View style={style.textContainer}>
-                    <View>
-                        <Text style={style.displayName}>
-                            {displayUsername(user, teammateNameDisplay)}
-                        </Text>
+            <View style={style.container}>
+                <CustomListRow
+                    id={id}
+                    theme={theme}
+                    onPress={this.onPress}
+                    enabled={enabled}
+                    selectable={selectable}
+                    selected={selected}
+                >
+                    <View style={style.profileContainer}>
+                        <ProfilePicture
+                            userId={id}
+                            size={32}
+                        />
                     </View>
-                    <View>
-                        <Text
-                            style={style.username}
-                            ellipsizeMode='tail'
-                            numberOfLines={1}
-                        >
-                            {usernameDisplay}
-                        </Text>
+                    <View style={[style.textContainer, (showTeammateDisplay ? style.showTeammateDisplay : style.hideTeammateDisplay)]}>
+                        <View>
+                            <Text
+                                style={style.username}
+                                ellipsizeMode='tail'
+                                numberOfLines={1}
+                            >
+                                {usernameDisplay}
+                            </Text>
+                        </View>
+                        {showTeammateDisplay &&
+                        <View>
+                            <Text
+                                style={style.displayName}
+                                ellipsizeMode='tail'
+                                numberOfLines={1}
+                            >
+                                {teammateDisplay}
+                            </Text>
+                        </View>
+                        }
                     </View>
-                </View>
-            </CustomListRow>
+                    <View style={{width: 25}}/>
```

Comment:
Above is just a filler on right side.  Don't know why adding margin or padding to the right/horizontal does not work for me.

Reply:
ok but make it a style